project-base/src/Shopsys/ShopBundle/Resources/views/Front/Inline/Newsletter/subscription.html.twig

```diff
@@ -1,4 +1,12 @@
-{{ form_start(form, { attr: { 'data-on-submit': 'ajaxSubmit', 'data-success': success ? 'true' : 'false' }}) }}
+{{ form_start(form, {
+    attr: {
+        'data-on-submit': 'ajaxSubmit',
+        'method': 'GET',
```

Comment:
Isn't that the default value?

Reply:
Also, does it even matter? I mean, it is only sent in JS and there the method is hard-coded.